dbms/src/Interpreters/Aggregator.h

```diff
@@ -718,7 +718,7 @@ struct AggregatedDataVariants : private boost::noncopyable
     case Type::NAME:                                                                                       \
     {                                                                                                      \
         const auto * ptr = reinterpret_cast<const AggregationMethodName(NAME) *>(aggregation_method_impl); \
-        return ptr->data.size() + (without_key != nullptr);                                                \
+        return ptr->data.size();                                                                           \
```

Comment:
why remove the (without_key != nullptr) check here?

Reply:
It is used by `over_flow_row`, and we already remove the support of `over_flow_row`